examples/rdkafka_example.c

```diff
@@ -187,6 +189,27 @@ static void msg_consume (rd_kafka_message_t *rkmessage,
 				!timestamp ? 0 :
 				(int)time(NULL) - (int)(timestamp/1000));
 		}
+
+                if (!rd_kafka_message_headers(rkmessage, &hdrs)) {
```

Comment:
some n00b commentary for your entertainment. I find this difficult to read, though I guess it's a Cism and so no problem.  I'd have written this here: `if (rd_kafka_message_headers(rkmessage, &hdrs) == RD_KAFKA_RESP_ERR_NO_ERROR)`.